### ISSUE TITLE
special treatment for DDL

### ISSUE THREAD

**Issue by reata** [reata/sqllineage#28]
Currently sqllineage treat every sql as DML. When there are DDLs, lineage result is weird.

```sql
create table taba like tabb;
alter table taba rename to tabb;
drop table if exists taba;
```

**Comment by reata** [comment]
another question is: If DDL is treated specially, of all the DML, should we treat SELECT command specially? After all, it's not like insert or insert overwrite.

**Comment by reata** [comment]
As of now, CREATE and DROP is already taken care of. To avoid significant change, let's add a processing logic for ALTER that's specific for table name renaming.

**Comment by reata** [comment]
Also truncate DDL needs special care.

**Comment by reata** [comment]
this should be closed by #44
